hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/Networking.java

```diff
@@ -17,33 +17,35 @@
 package com.hazelcast.jet.impl;
 
 import com.hazelcast.cluster.Address;
-import com.hazelcast.internal.nio.BufferObjectDataInput;
-import com.hazelcast.internal.nio.BufferObjectDataOutput;
 import com.hazelcast.internal.nio.Connection;
 import com.hazelcast.internal.nio.Packet;
 import com.hazelcast.jet.impl.execution.ExecutionContext;
 import com.hazelcast.jet.impl.execution.SenderTasklet;
+import com.hazelcast.jet.impl.serialization.MemoryDataInput;
+import com.hazelcast.jet.impl.serialization.DataInputFactory;
+import com.hazelcast.jet.impl.serialization.MemoryDataOutput;
+import com.hazelcast.jet.impl.serialization.DataOutputFactory;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.spi.impl.NodeEngine;
 import com.hazelcast.spi.impl.NodeEngineImpl;
 
-import java.io.IOException;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.ScheduledFuture;
 
 import static com.hazelcast.internal.nio.Packet.FLAG_JET_FLOW_CONTROL;
 import static com.hazelcast.internal.nio.Packet.FLAG_URGENT;
 import static com.hazelcast.jet.Util.idToString;
-import static com.hazelcast.jet.impl.util.ExceptionUtil.sneakyThrow;
-import static com.hazelcast.jet.impl.util.ImdgUtil.createObjectDataInput;
-import static com.hazelcast.jet.impl.util.ImdgUtil.createObjectDataOutput;
 import static com.hazelcast.jet.impl.util.ImdgUtil.getMemberConnection;
 import static com.hazelcast.jet.impl.util.ImdgUtil.getRemoteMembers;
 import static com.hazelcast.jet.impl.util.Util.uncheckRun;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
 public class Networking {
+
+    private static final int PACKET_HEADER_SIZE = 16;
+    private static final int FLOW_PACKET_INITIAL_SIZE = 128; // TODO:
```

Comment:
The default was 1 << 15... which seems a little bit excessive. Is there any sensible average I could default to?